packages/curriculum-compiler-json/__tests__/fixtures/exercise/sample/ast.json

```diff
@@ -0,0 +1,68 @@
+{
+  "type": "root",
+  "children": [
+    {
+      "type": "yaml",
+      "value":
+        "author: milesflo\n\nlevels:\n\n  - beginner\n\n  - basic\n\ntype: exercise\n\nlink: https://www.codewars.com/kata/classy-classes\nlinkType: codewars\nstandards:\n\n  javascript.execution-context.2: 1000\n  javascript.execution-context.3: 1000\n  javascript.execution-context.4: 1000\n\nlinks:\n\n  - '[MDN - this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this)'",
```

Comment:
both of these should be in the `data.parsedValue` object

Reply:
Well no, if you look at the `/insight/sample/ast.json`, it has a similar heirarchy of having the `value: <plaintext>` and `data.parsedValue: <json>`